modules/site-health/audit-enqueued-assets/load.php

```diff
@@ -0,0 +1,232 @@
+	 */
+	$enqueued_scripts = perflab_aea_get_total_enqueued_scripts();
+	if ( false === $enqueued_scripts ) {
+		return array();
+	}
+
+	$result = array(
+		'label'       => esc_html__( 'Enqueued scripts', 'performance-lab' ),
+		'status'      => 'good',
+		'badge'       => array(
+			'label' => esc_html__( 'Performance', 'performance-lab' ),
+			'color' => 'blue',
+		),
+		'description' => sprintf(
+		/* translators: 1: Number of enqueued scripts. 2: "script" word. 3.Scripts size. */
+			'<p>' . esc_html__( 'The amount of %1$s enqueued %2$s (size: %3$s) is acceptable.', 'performance-lab' ) . '</p>',
+			$enqueued_scripts,
+			_n( 'script', 'scripts', $enqueued_scripts, 'performance-lab' ),
+			size_format( perflab_aea_get_total_size_bytes_enqueued_scripts() )
+		),
+		'actions'     => '',
+		'test'        => 'enqueued_js_assets',
+	);
+
+	if ( $enqueued_scripts > 30 || perflab_aea_get_total_size_bytes_enqueued_scripts() > 300000 ) {
+		$result['status']         = 'recommended';
+		$result['badge']['color'] = 'orange';
+		$result['description']    = sprintf(
+		/* translators: 1: Number of enqueued scripts. 2: "script" word. 3.Scripts size. */
+			'<p>' . esc_html__( 'Your website enqueues %1$s %2$s (size: %3$s). Try to reduce the number or to concatenate them.', 'performance-lab' ) . '</p>',
+			$enqueued_scripts,
+			_n( 'script', 'scripts', $enqueued_scripts, 'performance-lab' ),
+			size_format( perflab_aea_get_total_size_bytes_enqueued_scripts() )
+		);
+		$result['actions'] .= sprintf(
+			/* translators: 1: HelpHub URL. 2: Link description. 3.URL to clean cache. 4. Clean Cache text. */
+			'<p><a target="_blank" href="%1$s">%2$s</a></p><p><a href="%3$s">%4$s</a></p>',
+			esc_url( __( 'https://wordpress.org/support/article/optimization/', 'performance-lab' ) ),
+			esc_html__( 'More info about performance optimization', 'performance-lab' ),
+			esc_url( add_query_arg( 'action', 'clean_aea_audit', wp_nonce_url( admin_url( 'site-health.php' ), 'clean_aea_audit' ) ) ),
+			esc_html__( 'Clean Test Cache', 'performance-lab' )
+		);
+	}
+
+	return $result;
+}
+
+/**
+ * Callback for enqueued_css_assets test.
+ *
+ * @since 1.0.0
+ *
+ * @return array
+ */
+function perflab_aea_enqueued_css_assets_test() {
+	/**
+	 * If the test didn't run yet, deactivate.
+	 */
+	$enqueued_styles = perflab_aea_get_total_enqueued_styles();
+	if ( false === $enqueued_styles ) {
+		return array();
+	}
+	$result = array(
+		'label'       => esc_html__( 'Enqueued styles', 'performance-lab' ),
+		'status'      => 'good',
+		'badge'       => array(
+			'label' => esc_html__( 'Performance', 'performance-lab' ),
+			'color' => 'blue',
+		),
+		'description' => sprintf(
+		/* translators: 1: Number of enqueued styles. 2: "styles" word. 3.Styles size. */
+			'<p>' . esc_html__( 'The amount of %1$s enqueued %2$s (size: %3$s) is acceptable.', 'performance-lab' ) . '</p>',
+			$enqueued_styles,
+			_n( 'style', 'styles', $enqueued_styles, 'performance-lab' ),
```

Comment:
See my other comment, this needs to be integrated into the translation string in line 172.